recipes/Florentiner.qmd

```diff
@@ -0,0 +1,42 @@
+---
+title: "The best Florentiners"
+image: "https://heissehimbeeren.com/wp-content/uploads/2022/11/omas-florentiner-rezept-12.webp"
+image-alt: "https://images.lecker.de/klassische-florentiner/1x1,id=72f9c685,b=lecker,w=1600,h=,ca=12.50,0,87.50,100,rm=sk.jpeg"
+categories:
+  - "Christmas"
+  - "Cookie"
+  - "Sweet"
+  - "Vegetarian"
+
+---
+
+## {{< fa basket-shopping >}} Ingredients
+
+For the shortcrust dough:
+- [175 g] soft butter
```

Comment:
For all ingredients, please change to, e.g.,:

```r
- [ ] 175 g soft butter
```

The square brackets should be empty and only contain a space. In Markdown syntax, this then shows up as To-Do-List, like so:

- [ ] 175 g soft butter